server/src/http/claims.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct JwtClaims {
```

Comment:
rest of the structs in this file starts with Http, perhaps this one should too?
`HttpJwtClaims`?

Reply:
I'd stick to JWT as its an acronym on its own :)